ludwig/models/llm.py

```diff
@@ -65,13 +65,13 @@ def __iter__(self) -> None:
         return iter(self.obj.keys())
 
     def keys(self) -> List[str]:
-        return self.obj.keys()
+        return self.obj.key_list()
```

Comment:
@dennisrall Do you think it might be simpler to retain the coding pattern of returning `list(my_object.keys())` or `list(my_object.items())`, etc., instead of introducing a special method `i.tem_list()`?  It seems to me that doing so will be consistent with other cases in these Ludwig modules as well as general Python collection patterns.  Thank you.